.github/workflows/build.yaml

```diff
@@ -194,8 +194,16 @@ jobs:
       matrix:
         ssllib: [ openssl11, openssl3, libressl]
         build: [ normal, asan ]
-        os: [macos-11, macos-12]
+        os: [macos-12, macos-13, macos-14]
         include:
+           # macos14 and newer runners use ARM CPUs and homebrew uses /opt/homebrew/
```

Comment:
do you update MacOS build simultaneously? Could you tell the purpose?

Reply:
I checked original repo and it seems to be different there. Could you summarize your changes?

Reply:
I think I looked at this particular tag, but OK. The most important is that it is not custom change.